build/config/PublishData.json

```diff
@@ -5,56 +5,28 @@
             "version": "2.8.*",
```

Comment:
This would update to 2.11 in sync with the other change. (And eventually 3.0 once we've scouted the other Willow concerns.)

Reply:
Line 5 now has `2.11.*` in it if you check the `Files changed` tab - not sure why this preview shows the old still.

Reply:
Hmm. Why is this line on the `dev15.7.x` line? That one is old and should stay the same, right?